airflow/executors/base_executor.py

```diff
@@ -129,6 +129,19 @@ def heartbeat(self):
         Stats.gauge('executor.queued_tasks', num_queued_tasks)
         Stats.gauge('executor.running_tasks', num_running_tasks)
 
+        self.trigger_tasks(open_slots)
+
+        # Calling child class sync method
+        self.log.debug("Calling the %s sync method", self.__class__)
+        self.sync()
+
+    def trigger_tasks(self, open_slots):
+        """
+        Trigger tasks. This function is overwritten in CeleryExecutor
```

Comment:
I think it is fine that you don't mention `This function is overwritten in CeleryExecutor`